src/conversion/configurationdefinedconverters.js

```diff
@@ -0,0 +1,310 @@
+ *		<h1>Another heading</h1>
+ *		<h1>Paragraph-like heading</h1>
+ *		<h1>Another non-semantic header</h1>
+ *
+ * @param {module:engine/conversion/configurationdefinedconverters~ConverterDefinition} definition A conversion configuration.
+ * @param {Array.<module:engine/conversion/viewconversiondispatcher~ViewConversionDispatcher>} dispatchers
+ */
+export function viewToModelElement( definition, dispatchers ) {
+	const { model: modelElement, viewDefinitions } = parseConverterDefinition( definition );
+
+	const converter = prepareViewConverter( dispatchers, viewDefinitions );
+
+	converter.toElement( modelElement );
+}
+
+/**
+ * Helper for creating model to view converter from model's attribute
+ * to {@link module:engine/view/attributeelement~AttributeElement AttributeElement}. The `acceptAlso` property is ignored.
+ *
+ * You can define conversion as simple model element to view element conversion using simplified definition:
+ *
+ *		modelAttributeToViewAttributeElement( 'bold', {
+ *			model: 'true',
+ *			view: 'strong',
+ *		}, [ dispatcher ] );
+ *
+ * Or defining full-flavored view object:
+ *
+ *		modelAttributeToViewAttributeElement( 'fontSize' {
+ *			model: 'big',
+ *			view: {
+ *				name: 'span',
+ *				styles: {
+ *					'font-size': '1.2em'
+ *				}
+ *			},
+ *		}, [ dispatcher ] );
+ *
+ * Above will generate an HTML tag for model's attribute `fontSize` with a `big` value set:
+ *
+ *		<span style="font-size:1.2em;">...</span>
+ *
+ * @param {String} attributeName Attribute name from which convert.
+ * @param {module:engine/conversion/configurationdefinedconverters~ConverterDefinition} definition A conversion configuration.
+ * @param {Array.<module:engine/conversion/modelconversiondispatcher~ModelConversionDispatcher>} dispatchers
+ */
+export function modelAttributeToViewAttributeElement( attributeName, definition, dispatchers ) {
+	const { model: attributeValue, viewDefinition } = parseConverterDefinition( definition );
+
+	buildModelConverter()
+		.for( ...dispatchers )
+		.fromAttribute( attributeName )
+		.toElement( value => {
+			if ( value != attributeValue ) {
+				return;
+			}
+
+			return AttributeElement.fromViewDefinition( viewDefinition );
+		} );
+}
+
+/**
+ * Helper for creating view to model converter from view to model attribute. It will convert also all matched view elements defined in
+ * `acceptAlso` property. The `model` property is used as model's attribute value to match.
+ *
+ * Conversion from model to view might be defined as simple one to one conversion:
+ *
+ *		viewToModelAttribute( 'bold', { model: true, view: 'strong' }, [ dispatcher ] );
+ *
+ * As a full-flavored definition:
+ *
+ *		viewToModelAttribute( 'fontSize', {
+ *			model: 'big',
+ *			view: {
+ *				name: 'span',
+ *				style: {
+ *					'font-size': '1.2em'
+ *				}
+ *			}
+  *		}, [ dispatcher ] );
+ *
+ * or with `acceptAlso` property to match many elements:
+ *
+ *		viewToModelAttribute( 'fontSize', {
+ *			model: 'big',
```

Comment:
I don't like the fact that attirbute name is passed as a separate parameter (what also @jodator notice in the PR description). It is also strange that in one case `model` is the model element name and in the other it is attibute value. 

At, the same time I undrestand that for feature configuration it is the best, because feature knows what is the name of the attribute.

However, I would go here with:

```js
model: {
   attributes: { 'fontSize': 'big' }
}
```

This is configuration of the conversion building tool. The plugin configuration does not need to be the same. Plugin can do some transformation adding obvious data to it, to make its configuration as simple as possible. However, this config will not fit all features need and we should not try to make it 1-1 with the config.